.zuul.d/project-templates.yaml

```diff
@@ -24,6 +25,7 @@
               - name: github.com/ansible-collections/servicenow.itsm
               - name: github.com/ansible-collections/splunk.es
               - name: github.com/ansible-collections/trendmicro.deepsec
+              - name: github.com/ansible-collections/vmware.vmware_rest
```

Comment:
We also need cloud.common.

Reply:
Cloud.common is already part of this file, added via https://github.com/ansible/network-ee/pull/65.